mmv1/templates/terraform/examples/firebase_android_app_basic.tf.erb

```diff
@@ -0,0 +1,20 @@
+resource "google_project" "default" {
```

Comment:
if we enable delete as above, we can avoid creating a new project each time.

Reply:
Done.